labels/mapping.go

```diff
@@ -35,8 +61,10 @@ var helpWanted = []Label{
 	{"Good First Issue", "Suitable for first-time contributors", "7057ff"},
 	{"Hacktoberfest", "", "dc7d02"},
 	{"Help Wanted", "Community help wanted", "33aa3f"},
-	{"Needs spec", "Needs further specification to be a good help wanted ticket", "88acea"},
+	// {"Needs spec", "Needs further specification to be a good help wanted ticket", "88acea"},
```

Comment:
We discussed that this may not be needed as a special label... Just a normal Help Wanted

Reply:
I agree. 0/5 if we need this.

Reply:
One can always create more labels in repos, and if this proves to be useful, we can add it to the list.